### PR TITLE
Related to Issue #2000. Updated Encoding parameter info for all Export-Clixml

### PR DESCRIPTION
**Pull request by davidsmatlak**
Related to Issue #2000 . Updated Encoding parameter info for all Export-Clixml. 

Style updates, paragraph reflows, fixed links.

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a2ca63a](https://github.com/davidsmatlak/PowerShell-Docs/pull/3590/commits/a2ca63a0cbdf184322e1f35182dcdf71cf6858f1)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a2ca63a](https://github.com/davidsmatlak/PowerShell-Docs/pull/3590/commits/a2ca63a0cbdf184322e1f35182dcdf71cf6858f1)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 16 sec(s)_

File: reference/6/Microsoft.PowerShell.Utility/Export-Clixml.md [hunk 1]
```diff
@@ -249,23 +273,29 @@ Accept wildcard characters: False
...
 ## OUTPUTS
 
 ### System.IO.FileInfo
-**Export-Clixml** creates a file that contains the XML.
+
+`Export-Clixml` creates a file that contains the XML.
 
 ## NOTES
 
 ## RELATED LINKS
```
Comment (davidsmatlak) [inline reference/6/Microsoft.PowerShell.Utility/Export-Clixml.md]: There were two preexisting external links in the documents. Should they be removed?

Comment (davidsmatlak) [reply reference/6/Microsoft.PowerShell.Utility/Export-Clixml.md]: Confirmed that the links are acceptable.

**Pull request merged**